### DISCUSSION TITLE
salvataggio nuovo Cliente contestuale a creazione Fattura

### DISCUSSION THREAD

**Discussion by zofe** [fattureincloud/api#79] [Q&A] [answered]
Era possibile con le vecchie API salvare il cliente contestualmente alla generazione della prima fattura,
E' una cosa ancora possibile con l'SDK?

**Comment by emajo** [comment, marked as answer]
Ciao, @zofe come spiegato [qui](https://developers.fattureincloud.it/docs/api-v2-migration#-documenti-emessi) le api v2 per ora non supportano la funzionalità `salva_anagrafica`, ma verrà implementata in futuro come puoi vedere dalla nostra [roadmap](https://trello.com/b/uAjGCOhL/fic-api-roadmap-overview). Attualmente è necessario creare il cliente attraverso [l'apposita chiamata](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/entities/clients) prima di creare la fattura.

Resto a disposizione per ulteriori chiarimenti,
Emanuele
